src/lib/rng/system_rng/system_rng.cpp

```diff
@@ -28,39 +28,39 @@ namespace Botan {
 
 namespace {
 
-class System_RNG_Impl : public RandomNumberGenerator
+class System_RNG_Impl final : public RandomNumberGenerator
    {
    public:
       System_RNG_Impl();
       ~System_RNG_Impl();
 
-      void randomize(byte buf[], size_t len) override;
-
       bool is_seeded() const override { return true; }
+
       void clear() override {}
-      std::string name() const override { return "system"; }
 
-      size_t reseed_with_sources(Entropy_Sources&,
-                                 size_t /*poll_bits*/,
-                                 std::chrono::milliseconds /*timeout*/) override
-         {
-         // We ignore it and assert the PRNG is seeded.
-         // TODO: could poll and write it to /dev/urandom to help seed it
-         return 0;
-         }
+      void randomize(Botan::byte out[], size_t len) override;
```

Comment:
This is used in some places to avoid unfortunate `byte` typedef collisions from the global namespace, though I'm not sure if it was really necessary here. It would be better to use `uint8_t` instead. (Eventually I will mass search and replace this, when PR activity slows down a bit)